app/controllers/review_item/reject.js

```diff
@@ -1,24 +1,24 @@
-import Ember from 'ember';
+import Ember from "ember";
 import { translationMacro as t } from "ember-i18n";
 const { getOwner } = Ember;
 
 export default Ember.Controller.extend({
-
   reviewItem: Ember.inject.controller(),
-  reviewOfferController: Ember.inject.controller('review_offer'),
+  reviewOfferController: Ember.inject.controller("review_offer"),
   offer: Ember.inject.controller(),
 
-  itemTypeId: Ember.computed.alias('reviewItem.itemTypeId'),
-  itemId: Ember.computed.alias('reviewItem.model.id'),
-  rejectionReasonId: Ember.computed.alias('model.rejectionReason.id'),
+  itemTypeId: Ember.computed.alias("reviewItem.itemTypeId"),
+  itemId: Ember.computed.alias("reviewItem.model.id"),
+  rejectionReasonId: Ember.computed.alias("model.rejectionReason.id"),
   rejectReasonPlaceholder: t("reject.custom_reason"),
   messageBox: Ember.inject.service(),
   i18n: Ember.inject.service(),
   itemPackages: Ember.computed.alias("item.packages"),
+  rejectMsg: "",
 
-  rejectReason: Ember.computed('itemId', {
+  rejectReason: Ember.computed("itemId", {
     get: function() {
-      return this.get('reviewItem.model.rejectReason');
+      return this.get("reviewItem.model.rejectReason");
     },
     set: function(key, value) {
       return value;
```

Comment:
this setter isn't setting anything 😄 , we can remove it

Reply:
Good catch @patrixr. :)